sandbox/tests/storage_test.rs

```diff
@@ -34,8 +35,40 @@ fn call_in_wasm<E: Externalities> (
     )
 }
 
-// TODO Storage IT Tests
-
 #[test]
 fn test_ext_storage_get(){
-}
\ No newline at end of file
+    let setup = Setup::new();
+    let mut ext = setup.ext;
+    let mut ext = ext.ext();
+    ext.set_storage(b"python".to_vec().encode(), b"greatest".to_vec()); 
```

Comment:
You can extract the key and value into separate variables, set them in the storage and then once you have to assert you can simply assert the storage `value` with the result.
My thinking is that:
1) It will save you the redundant "python" usage 
2) you will not have to do `.encode` twice for the `key` and once for the `value` as you do now